src/Card/SummaryCard/SummaryCard.tsx

```diff
@@ -158,8 +153,8 @@ function SummaryCard({
       <Card sx={{ height, width }}>
         <CardHeader
           sx={{
-            "& .MuiCardHeader-content": {
-              width: "100%"
+            [` .${cardHeaderClasses.content}`]: {
+              overflowX: "hidden"
             },
```

Comment:
Rather than forcing a pixel width, we can make use of the fact the underlying card is flex, and force it to not overflow horizontally. This leaves the title and subtitle components to use the full available width whether there is an action button present or not.

With the previous pixel width based approach, I ended up with unused white space if my text overflowed, and I didn't have a menu.